packages/anu-vue/test/setup.vitest.ts

```diff
@@ -1,7 +1,10 @@
-import { afterAll, beforeAll } from 'vitest'
+import { afterAll, beforeAll, vi } from 'vitest'
 import { createApp } from 'vue'
+import ResizeObserver from 'resize-observer-polyfill'
 import { plugin } from '../src/plugin'
 
+vi.stubGlobal('ResizeObserver', ResizeObserver)
```

Comment:
Why do we need this, I probably seen this somewhere before 🤔 ?

Reply:
floating-ui depends on this API, but it does not exist in the Node.js environment.